### PR TITLE
Related with DROTH-860

### PR DESCRIPTION
**Pull request by joanacoelho**

### CODE REVIEW

**Comment by marquesfm** [comment]
Ok for me

File: digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala [hunk 1]
```diff
@@ -348,14 +350,12 @@ Returns empty result as Json message, not as page not found
       halt(BadRequest("Invalid property values on: " + propertiesWithInvalidValues.map(_.publicId).mkString(", ")))
   }
 
-  private def validateBusStopDirections(properties: Seq[SimpleProperty], linkId: Long) = {
-    val roadLink = roadLinkService.getRoadLinkAndComplementaryFromVVH(linkId)
-    val roadLinkDirection = roadLink.map(dir => dir.trafficDirection).headOption
-    val massDirection = massTransitStopService
+  private def validateBusStopDirections(properties: Seq[SimpleProperty], roadLinks: Option[RoadLink]) = {
+    val roadLinkDirection = roadLinks.map(dir => dir.trafficDirection).headOption
```
Comment (MAC-CGI) [inline digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala]: Maybe this method don't need to be changed

File: digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala [hunk 2]
```diff
@@ -288,10 +288,12 @@ Returns empty result as Json message, not as page not found
     }
     val (optionalLon, optionalLat, optionalLinkId, bearing) = massTransitStopPositionParameters(parsedBody)
     val properties = (parsedBody \ "properties").extractOpt[Seq[SimpleProperty]].getOrElse(Seq())
-    val linkId = (parsedBody \ "linkId").extractOpt[Long]
+    val id = params("id").toLong
+    val persistedAsset = massTransitStopService.getPersistedAssetsByIds(Set(id)).headOption
+    val roadLinks = persistedAsset.flatMap { asset => roadLinkService.getRoadLinkAndComplementaryFromVVH(asset.linkId) }
```
Comment (MAC-CGI) [inline digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala]: This val roadLinks =  persistedAsset. ... coulde be removed and send to validateBusStopDirections(properties, asset.linkId)

File: digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala [hunk 3]
```diff
@@ -370,7 +370,8 @@ Returns empty result as Json message, not as page not found
     validateUserRights(linkId)
     validateBusStopMaintainerUser(properties)
     validateCreationProperties(properties)
-    validateBusStopDirections(properties, linkId)
+    val roadLinks = roadLinkService.getRoadLinkAndComplementaryFromVVH(linkId)
```
Comment (MAC-CGI) [inline digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala]: Maybe the code could remain the same as it was

**Pull request closed without merging**